### DISCUSSION TITLE
Copilot not working on corporate Macbook (M1)

### DISCUSSION THREAD

**Discussion by qazt482** [community/community#41174] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I am getting an error

`[ERROR] [default] [2022-12-09T10:02:36.293Z] GitHub Copilot could not connect to server. Extension activation failed: "self signed certificate in certificate chain"`

Searching for a solution only takes me to Windows and using the Win-ca extension.

What do I do to fix this on a mac?

**Comment by Akash1134** [comment]
Hi @qazt482 , The error `Extension activation failed: self signed certificate in certificate chain` occurs in a corporate setup as your network intercepts the SSL traffic which leaves behind a security certificate chain that Copilot is not able to trust. You can use this [marketplace extension](https://marketplace.visualstudio.com/items?itemName=linhmtran168.mac-ca-vscode) to allow Copilot to trust the CA.  Hope that helps!
